pages/product/[code].js

```diff
@@ -183,9 +236,32 @@ export default function DetailPage() {
         <>
           <StyledAllCards>
             <StyledProductCard>
-              <StyledProductName>
-                {data.product.product_name}{" "}
-              </StyledProductName>
+              <StyledProductName>{data.product.product_name}</StyledProductName>
+              <StyledButtonDiv>
+                <StyledSaveButton
```

Comment:
please be explicit about the type of button
```
type="button"
```

Reply:
ok i will !! :) thanks